ansible-ipi-install/inventory/hosts.sample

```diff
@@ -141,6 +141,9 @@ pullsecret=""
 #provisioningHostIP=<baremetal_net_IP1>
 #bootstrapProvisioningIP=<baremetal_net_IP2>
 
+# (Optional) Change the boot mode of the OpenShift cluster nodes to legacy mode (BIOS). Default is UEFI. 
```

Comment:
Can you add a comment saying, if this is commented or set to anything other than legacy, UEFI is used.